src/js/views/createCertificate/translations/en.createCertificate.i18n.json

```diff
@@ -0,0 +1,36 @@
+{
+  "headerTitle": "New Certificate",
+  "finishButton": "Finish",
+  "cancelButton": "Cancel",
+  "createCertificateOneClick": {
+    "title": "Create certificate with one click",
+    "subTitle": "More practical: The platform generates the certificate and private key for the device"
+  },
+  "createCertificateCSR": {
+    "title": "Submit a Certificate Signing Request (CSR)",
+    "subTitle": "More secure: The platform generates a certificate but has no knowledge of the private key",
+    "generateCsrHelp": "How to generate a CSR ?",
+    "csrHelpSteps": {
+      "step1": "Step 1",
+      "step1_text": "Run the command openssl req  -newkey 'rsa:2048'  -nodes  -sha256  -keyout 'privatekey.pem'  -out 'csr.pem'  -subj '/CN=Any value'",
```

Comment:
1. Use calmelCase, not snake_case, because i18next use _ to its features like `_plural`
2. I think there is two spaces between the command arguments